### PR TITLE
ProduceWordBags Onnx Export Fix 

### PR DESCRIPTION
**Pull request by Lynx1820**
`ProduceWordBags` Onnx export assumes that the input will be a string of untokenized text. However, in ML.NET the user can choose to tokenize the text before passing it. Even though this is an unnecessary step, since a `WordTokenizer` is added before the `NgramExtraction` transformer, there is no error in ML.NET but there is an error in OnnxRuntime because a double tokenization resulted in more dimensions in Onnx. I have changed the Squeeze transformer to Reshape, so the dimension can remain consistent in this edge case.

### CODE REVIEW

File: test/Microsoft.ML.Tests/OnnxConversionTest.cs [hunk 1]
```diff
@@ -1310,22 +1310,25 @@ public void WordTokenizerOnnxConversionTest()
             IEstimator<ITransformer>[] pipelines =
             {
                 mlContext.Transforms.Text.TokenizeIntoWords("Tokens", "Text", new[] { ' ' })
-                                .Append(mlContext.Transforms.Conversion.MapValueToKey("Tokens"))
-                                .Append(mlContext.Transforms.Text.ProduceNgrams("NGrams", "Tokens",
-                                            ngramLength: ngramLength,
-                                            useAllLengths: useAllLength,
-                                            weighting: weighting)),
+                .Append(mlContext.Transforms.Conversion.MapValueToKey("Tokens"))
```
Comment (harishsk) [inline test/Microsoft.ML.Tests/OnnxConversionTest.cs:1313]: Any particular reason the formatting was changed? Usually there is a small offset to indicate line continuation.

Comment (Lynx1820) [reply test/Microsoft.ML.Tests/OnnxConversionTest.cs:1313]: No, I noticed that line 1321 was not aligned with the rest. I'll add the offsets back though and just align that line.

Comment (harishsk) [reply test/Microsoft.ML.Tests/OnnxConversionTest.cs:1313]: nit: It looks like the offset for line 1313 is still wrong. It seems to be left aligned with the previous line.

**Review by harishsk — APPROVED** [review]
Please fix the formatting.
